packages/sanity/src/core/form/inputs/PortableText/toolbar/BlockStyleSelect.tsx

```diff
@@ -136,6 +136,7 @@ export const BlockStyleSelect = memo(function BlockStyleSelect(
         mode="bleed"
         onClick={preventDefault}
         text={menuButtonText}
+        justify="flex-start"
```

Comment:
I think we should make sure the caret is always right-aligned, so that it doesn't change position when the text on the left changes.

```suggestion
        justify="space-between"
```

Reply:
Thank you @sjelfull , this makes sense 🙇